modules/serverless-creating-subscriptions-YAML.adoc

```diff
@@ -3,8 +3,8 @@
 // <List assemblies here, each on a new line>
 // * serverless/event_workflows/serverless-channels.adoc
 
-[id="serverless-creating-subscriptions_{context}"]
-= Creating a subscription
+[id="serverless-creating-subscriptions-YAML_{context}"]
```

Comment:
Similar to ODC recommendation, I'd lowercase YAML in header ID/file name

Reply:
done